docs/Contributing/stable-branch.rst

```diff
@@ -14,6 +15,8 @@ occasions where we are introducing major breaking changes that take time to reso
 ``stable`` branch. As development occurs on the ``master`` branch, there are going to be times when we want to
 back-port a commit (a single patch) or a Pull Request (a collection of commits) over to the ``stable`` branch.
 
+The ``stable`` branch is created when preparing for the first Release Candidate towards a stable release, see
+:doc:`release`.
 This page documents the rules and procedures for maintaining the ``stable`` branch.
```

Comment:
This is not part of the PR, but since you are in there would you change to 

This page documents the rules and procedures for the release process, which includes some maintenance activities for the ``stable`` branch.

Reply:
I don't think your sentence fits the page about `stable`. I added a mention of `stable` branch in The Release Process.

Reply:
Looks good.